core/src/main/java/hudson/cli/ListChangesCommand.java

```diff
@@ -22,10 +22,9 @@
  *
  * @author Kohsuke Kawaguchi
  */
+
 @Restricted(NoExternalUse.class) // command implementation only
-@Extension
```

Comment:
This line needs to be restored.  As far as I can tell from [list of extensions](https://jenkins.io/doc/developer/extensions/jenkins-core/), the ListChangesCommand class is annotated as a command line interface extension. By removing this line, the `list-changes` command will be deleted from the Jenkins command line interface.  That will break compatibility for all users of the `list-changes` command.

I doubt that it can be restored "as is", since I don't think you can annotate an abstract class with that annotation.  However, I have never tried that, so can't be sure.

Reply:
Thanks for the advice. The build was successful on jenkins on my local machine with this branch, so I believed the tests passed. Sorry for the trouble